manifests/repo.pp

```diff
@@ -25,7 +29,7 @@
         architecture => 'amd64',
         comment      => $description,
         location     => 'https://apt.releases.hashicorp.com',
-        release      => 'stable',
+        release      => $facts['os']['distro']['codename'],
```

Comment:
This should be done by default, right? https://github.com/puppetlabs/puppetlabs-apt/blob/c5259f83ad20d120d502d8541c75f4ee6021fec8/manifests/source.pp#L75-L83
```suggestion
```

Reply:
fixed in 44a46657826230706a1d05ab92f4d0c391b9f457